sql/synth_recipes.sql

```diff
@@ -2029,6 +2029,13 @@ INSERT INTO `synth_recipes` VALUES (25514,0,0,0,0,118,0,0,0,0,0,4096,4238,3323,8
 -- INSERT INTO `synth_recipes` VALUES (25527,0,2004,0,0,255,0,0,0,0,0,4096,4238,4076,9250,16408,0,0,0,0,0,21513,21514,21514,21514,1,1,1,1,'Raetic Baghnakhs');
 -- INSERT INTO `synth_recipes` VALUES (25528,0,0,0,255,255,0,0,0,0,0,4096,4238,4077,4081,9248,16605,0,0,0,0,21606,21607,21607,21607,1,1,1,1,'Enriching Sword');
 -- INSERT INTO `synth_recipes` VALUES (25529,0,2005,0,0,255,0,255,0,0,0,4102,4244,9003,9062,9251,9832,9844,9845,9845,9852,21628,21629,21630,21630,1,1,1,1,'Valor Sword');
+INSERT INTO `synth_recipes` VALUES (29886,1,0,0,0,71,0,0,0,0,0,4100,4242,12802,0,0,0,0,0,0,0,914,851,931,745,1,1,1,1,'Gold Cuisses (desynth)');
+INSERT INTO `synth_recipes` VALUES (29887,1,0,0,0,77,0,0,0,0,0,4100,4242,12674,0,0,0,0,0,0,0,850,931,931,745,2,2,3,1,'Gold Gauntlets (desynth)');
```

Comment:
I think desynth recipe is incorrect. The item requires 2 cermet chunk, but desynth rewards 2, then 3 cermet chunks.
It would be logical if it would be 1, then 2 for HQ1 and HQ2, no?

Reply:
Yea, I think that is a fair assessment.  I was just using what was listed on Wiki as some desynths do not seem to make much sense.